english/valarmorghulis.md

```diff
@@ -0,0 +1,4 @@
+A guy appears and replies with "valar dohaeris" and gives you a small coin.
+You admire the coin for a while and when you realize, the man disappeared.
+
+With no hope, but with a coin in the pocket, the only logical path is [here](../use-the-fork-luke/use-the-fork.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.